sql-select/actor-forty.sql

```diff
@@ -0,0 +1,4 @@
+select *
+  from "actors"
+  where "actorId" = 40
+  limit 1;
```

Comment:
Since the `where` clause is filtering by a unique column (there can't be more than one row with the same `"actorId"`), the `limit` isn't strictly necessary. Doesn't really hurt, though.